### ISSUE TITLE
Don't start ScopeThread's until UI is initialized

### ISSUE THREAD

**Issue by azonenberg** [glscopeclient/scopehal-apps#231]
With high-latency instruments, there's contention between waveform downloading and all of the initial queries populating instrument settings from cache.

Would be best if the instrument were kept on hold until the GUI had fully loaded.

**Comment by azonenberg** [comment]
ngscopeclient does background loads of stuff as much as possible, so I think we're probably good. Can make tweaks to individual drivers in the future as needed.
